src/collective/cover/browser/scaling.py

```diff
@@ -158,10 +158,12 @@ def modified(self):
                                             name='images',
                                             default=None)
             return base_scales and base_scales.modified()
-        mtime = ''
+        mtime = None
         for k, v in self.context.data.items():
             if INamedImage.providedBy(v):
-                mtime += self.context.data.get('{0}_mtime'.format(k), '')
+                image_time = self.context.data.get('{0}_mtime'.format(k), None)
+                if mtime is None or image_time > mtime:
```

Comment:
AFAIK, there are no tiles with more than one image field, so let's not solve problems we don't have; anyway all that code is overly complex and has to be completely refactored at some point.